crypto/err/err.c

```diff
@@ -584,7 +581,12 @@ char *ERR_error_string(unsigned long e, char *ret)
 
 LHASH_OF(ERR_STRING_DATA) *ERR_get_string_table(void)
```

Comment:
Huh. And the call within `openssl errstr` is just to implement the `-stats` option which seems not very useful and probably ought to be deleted.